jans-cedarling/bindings/cedarling_python/src/config/off_log_config.rs

```diff
@@ -0,0 +1,56 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use pyo3::prelude::*;
+
+/// OffLogConfig
+/// ============
+///
+/// `OffLogConfig` is a Python wrapper around the Rust `cedarling::LogTypeConfig` struct.
```

Comment:
I think `DisabledLoggingConfig` or `NoLoggingConfig` should be more descriptive

Reply:
In the documentation [link](https://github.com/JanssenProject/jans/wiki/Cedarling-Nativity-Plan)
We have `CEDARLING_LOG_TYPE` and one of possible value is `off`.

So I believe that the name of the config should be as close as possible to the version in the documentation

Reply:
Yes, but that's the name of _one_ of the variants, not the struct itself.

Reply:
Renamed to the `DisabledLoggingConfig`